src/index.css

```
@@ -5,6 +5,33 @@
     color-scheme: light dark;
+    --surface-base: 255 255 255;
+    --surface-card: 250 250 250;
+    --surface-glass: 255 255 255;
+    --surface-overlay: 255 255 255;
+    --line-default: 229 231 235;
+    --line-glass: 255 255 255;
+    --input-bg: 255 255 255;
+    --input-border: 209 213 219;
+    --input-text: 17 24 39;
+    --input-placeholder: 107 114 128;
+    --shadow-card: 0 10px 30px rgba(15, 23, 42, 0.08);
```

Stylelint (17.2.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)